src/components/button/README.md

```diff
@@ -42,7 +49,7 @@ Example:
 
 Output:
 
-<img src="https://material.angularjs.org/material2_assets/buttons/basic-buttons.png">
+![Basic Button Preview](https://cloud.githubusercontent.com/assets/4987015/13949035/8154175e-f023-11e5-9586-b6545dd60ba5.png)
```

Comment:
Can you update the button asset here instead?

https://github.com/angular/code.material.angularjs.org/blob/master/material2_assets/buttons/basic-buttons.png